tools/run-tests-within-container

```diff
@@ -8,7 +8,7 @@ TESTS_FAILED_FLAG=0
 run_test() {
     # Allow command line options without quotes
     # shellcheck disable=SC2086
-    prove -l ${PROVE_ARGS} "$@" || TESTS_FAILED_FLAG=1
+    prove -l ${PROVE_ARGS:+$PROVE_ARGS} "$@" || TESTS_FAILED_FLAG=1
```

Comment:
What is the point of the `+$PROVE_ARGS` part?

Reply:
it ensures that expands to nothing at all without quotes. for some reason the `${PROVE_ARGS:+}` when `$PROVE_ARGS=foo` still evaluates to nothing. 

What I am looking for is
- PROVE_ARGS="-v" make run-tests-within-container expands to ` prove -l -v -r`
- make run-tests-within-container expands to ` prove -l -r`
- PROVE_ARGS="" make run-tests-within-container expands to ` prove -l -r`
- and in no case expansion to `prove -l ' ' -r`


I used `${PROVE_ARGS:-}` also before but this complains about `SC2086 (info): Double quote to prevent globbing and word splitting.` 

how would you do it?

Reply:
Just look one line above!
I even mentioned this line yesterday.
You just need to copy this line to the one you changed.

Reply:
`# shellcheck disable=SC2086`